lib/module/interject.go

```diff
@@ -0,0 +1,27 @@
+package module
+
+import (
+	"fmt"
+	"github.com/dvdmuckle/irkbot/lib/configure"
+	"github.com/dvdmuckle/irkbot/lib/message"
+	"strings"
+)
+
+func HelpInterject() []string {
+	// This function returns an array of strings describing this command's
+	// functionality. It is displayed when someone gives the help command in a
+	// channel or private message.
+	s := "interject <subject> - return a popular copypasta defaulting to our favorite operating system"
+	return []string{s}
+}
+
+func Interject(cfg *configure.Config, in *message.InboundMsg, actions *Actions) {
+	var interMsg string
+	if len(in.MsgArgs[1:]) > 0 {
+		interMsg = strings.Join(in.MsgArgs[1:], " ")
+	} else {
+		interMsg = "Linux"
+	}
+	msg := fmt.Sprintf("I'd just like to interject for a moment. What you're refering to as %s, is in fact, GNU/%s, or as I've recently taken to calling it, GNU plus %s.", interMsg, interMsg, interMsg)
```

Comment:
Oh, also, I think that the `GNU/` should be changed to `GNU *slash*`, and `GNU plus` should be changed to `GNU *plus*`. I think that better fits the intended emphases.